packages/mdc-dialog/foundation.ts

```diff
@@ -69,6 +69,7 @@ export class MDCDialogFoundation extends MDCFoundation<MDCDialogAdapter> {
   private scrimClickAction_ = strings.CLOSE_ACTION;
   private autoStackButtons_ = true;
   private areButtonsStacked_ = false;
+  private suppressedPressSelector: string = strings.SUPPRESS_DEFAULT_PRESS_SELECTOR;
```

Comment:
nit: you can remove `: string`, typescript will infer the correct type since you assign it a string type